Code Review PR

teke85 opened the pull request:
The purpose of this PR is to allow my partner to perform a code review on my previous TO DO LIST Project.

Here are Steps I have taken to prepare for this task.

- Create a new empty repo.
- Copy my code from the "To Do list" project and commit it as one big chunk of code in a feature branch.
- Open a pull request.
- Ask my partner for a review in [GitHub](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review).
- Share the pull request link with my partner.

teke85 committed:
initial commit

Botlhale-Setou commented:
### Hey Teke 👋 
<br>

![my-man](https://user-images.githubusercontent.com/11477535/193818445-6de34c47-3ddc-436b-87c5-0b74d87b4bc3.gif)
<br>
**Great work on your project, it looks amazing bro!** The transitions are also a nice cherry on top 🍒 

✅ Well written README file. 
✅ Aesthetically pleasing design/theme.
✅ Great input-error handling.
✅ Local storage utilized well.
✅ All buttons work as they should.
<hr>

#### A few suggestions:

- [ ] ESLint returns errors due to the lack of semi-colons at the end of your instructions. Something like `npx eslint . --fix` should be able to remedy this quickly. <br><br>

- [ ] Consider **removing the scrollbars** from the `.list-item` class as they don't seem to be functional. Having them there without reason unnecessarily complicates the design. <br><br> ![image](https://user-images.githubusercontent.com/11477535/193821189-d110e6fa-b10d-4d6f-a3f0-c41fa1919865.png)
<br>

teke85 committed:
Fix linter issues

teke85 committed:
remove scroll bar

teke85 commented:
Hi @Botlhale-Setou thank you for taking the time to review my project. I fixed my linter issues and removed the scrollbars as advised. Thank you 
![To-Do List](https://user-images.githubusercontent.com/29442846/193864817-9223f4b7-6d73-469a-bbea-ed68331b3b71.png)

Botlhale-Setou commented:
### There's nothing else to say other than...
![perfect](https://user-images.githubusercontent.com/11477535/193866148-c3527eb4-2650-456d-a589-af13cad53e5e.gif)

<br>

**Go ahead and merge it!**
🥳 🥳 🥳
